src/server/transaction/server/driver.go

```diff
@@ -53,7 +53,7 @@ func (d *driver) batchTransaction(ctx context.Context, req []*transaction.Transa
 		// there is no need to persist the TransactionInfo to the collection
 		info := &transaction.TransactionInfo{
 			Transaction: &transaction.Transaction{
-				ID: uuid.New(),
+				ID: uuid.NewWithoutDashes(),
```

Comment:
Since we use the transaction ID as the commit ID, they need to be the same format